localstack-core/localstack/constants.py

```diff
@@ -11,6 +11,8 @@
 HEADER_LOCALSTACK_AUTHORIZATION = "x-localstack-authorization"
 HEADER_LOCALSTACK_TARGET = "x-localstack-target"
 HEADER_AMZN_ERROR_TYPE = "X-Amzn-Errortype"
+# HTTP headers used to override internal SQS ReceiveMessage
+HEADER_LOCALSTACK_SQS_OVERRIDE_MESSAGE_COUNT = "x-localstack-sqs-override-message-count"
```

Comment:
i think it would be better to move this into the SQS service module. having service-specific constants in the global constants file may clutter it over time.